src/main/java/com/google/devtools/build/lib/windows/WindowsFileSystem.java

```diff
@@ -148,31 +148,32 @@ protected FileStatus stat(PathFragment path, boolean followSymlinks) throws IOEx
       throw new FileNotFoundException(path + ERR_NO_SUCH_FILE_OR_DIR);
     }
 
-    final boolean isSymbolicLink = !followSymlinks && fileIsSymbolicLink(file);
-    final long lastChangeTime =
-        WindowsFileOperations.getLastChangeTime(getNioPath(path).toString(), followSymlinks);
     FileStatus status =
         new FileStatus() {
+          volatile Boolean isSymbolicLink;
```

Comment:
Let's mark this `@Nullable` and append a comment `// null if not yet known` to make the convention clear.